pretend.py

```diff
@@ -107,9 +108,18 @@ def __repr__(self):
         return "<call(%s%s%s)>" % (args, comma, kwargs)
 
 
-def call_recorder(func):
+def call_recorder(func, real_func=None, real_method=None):
+    if real_func is not None:
+        signature = inspect.signature(real_func)
+    if real_method is not None:
+        signature = inspect.signature(real_method)
+
     @functools.wraps(func)
     def inner(*args, **kwargs):
+        if real_func is not None:
+            signature.bind(*args, **kwargs)
+        if real_method is not None:
+            signature.bind_partial(None)
```

Comment:
Should this be `signature.bind(None, *args, **kwargs)` or something? Otherwise I don't see how this is doing anything.